contracts/LinearPremiumPriceOracle.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity >=0.5.0;
+
+import "./SafeMath.sol";
+import "./StablePriceOracle.sol";
+
+contract LinearPremiumPriceOracle is StablePriceOracle {
+    using SafeMath for *;
+
+    uint GRACE_PERIOD = 90 days;
+
+    uint public initialPremium;
+    uint public premiumDecreaseRate;
+
+    constructor(DSValue _usdOracle, uint[] memory _rentPrices, uint _initialPremium, uint _premiumDecreaseRate) public
+        StablePriceOracle(_usdOracle, _rentPrices)
+    {
+        initialPremium = _initialPremium;
+        premiumDecreaseRate = _premiumDecreaseRate;
+    }
+
+    function _premium(string memory name, uint expires, uint /*duration*/) internal view returns(uint) {
```

Comment:
Umm, looks like you now changed it to private function. What do we do if we want to show how much premium it is currently under (my demo was actually calling this function as well) ?